curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6145f59029474c0d3dc1c8b8.md

```diff
@@ -11,13 +11,26 @@ As with the other `input` and `label` elements, link the `textarea` to its corre
 
 # --hints--
 
-You should give the `label` element a `for` attribute.
+You should give the `textarea` element an `id` attribute.
 
 ```js
-assert.notEmpty(document.querySelectorAll('.formrow > .question-block')?.[1]?.querySelector('label')?.htmlFor);
+assert.notEmpty(document.querySelectorAll('.answer')?.[1]?.querySelector('textarea')?.id);
 ```
 
-You should give the `textarea` element an `id` attribute matching the `for` attribute of the `label` element.
+You cannot include spaces in the `id` for the `textarea` element.
```

Comment:
```suggestion
You should not include spaces in the `id` for the `textarea` element.
```

Reply:
A little more friendly in my opinion.

Reply:
Agreed, your version is friendlier. I was trying to convey the point that spaces are strictly not allowed in the `id` attribute, so I used "cannot" to emphasize this, but perhaps that isn't as important as being a little more user friendly. I think what I would really like to do is include an extra sentence explaining that the HTML specs don't allow spaces in the `id`, so that it is clear it's not just fCC imposing this restriction. Perhaps something like:

> You should not include spaces in the `id` for the `textarea` element. HTML does not allow spaces in the `id` attribute.

What do you think?

Reply:
Thank you for being so thoughtful Bruce. I do like teaching Campers this, but I think we should prevent spoiling the answer in step 40. 

Maybe there is a better opportunity in a different challenge. As for the provided sentence, I would not change anything.